visualization/fov/metadata_downloader.py

```diff
@@ -0,0 +1,174 @@
+from astropy.io import fits
+import astropy.units as u
+
+import numpy as np
+
+from astropy.coordinates import SkyCoord
+
+from sunpy.coordinates import frames
+from astropy.time import Time
+import scipy.io as sio
+from scipy.io import readsav
+import urllib.request
+
+
+def download_metadata(xrt_downloaded_files, filen, overwrite=False):
+    url_lis = xrt_downloaded_files[0][:]['fileid']
+    url_str_lis = []
+    for i in range(len(url_lis)):
+        url_str_lis.append(url_lis[i])
+    primary_hdu = fits.PrimaryHDU(data=np.ones((3, 3)))
+    c1 = fits.Column(name='URL', array=url_str_lis, format='100A')
+    c2 = fits.Column(name='header_int', array=np.asarray(range(len(url_str_lis)))+2, format='J')
+    table_hdu = fits.BinTableHDU.from_columns([c1, c2])
+
+    hdul2 = fits.HDUList([primary_hdu, table_hdu])
+    #hlis = []
+
+    for i in range(len(url_str_lis)):
+        if (i%10 == 0):
+            print(int(1000.0*i/len(url_str_lis))/10.0,'%')
+        fsspec_kwargs = {"block_size": 100_000, "cache_type": "bytes"}
+        with fits.open(url_lis[i], use_fsspec=True, fsspec_kwargs=fsspec_kwargs) as hdul:
+            #hlis.append(hdul[0].header)
+            # Download a single header
+            t_header = hdul[0].header
+            image_hdu = fits.ImageHDU(data=np.ones((100, 100)), header=t_header,name="header"+str(i))
+        hdul2.append(image_hdu)        
+    return hdul2
+
+    #hdul2.writeto(filen,overwrite=overwrite)
+
+def date_to_meta(xrt_download_list):
```

Comment:
```suggestion
def _date_to_meta(xrt_download_list):
```
If there's a function that is not intended for end users, then we can start the name with an underscore to indicate that it is private.